CMakeLists.txt

```diff
@@ -0,0 +1,150 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2013-2022 The Foundry Visionmongers Ltd
+
+#-----------------------------------------------------------------------
+# CMake preamble.
+
+# FindPython's Development.Module component was added in 3.18, and
+# pybind11 recommends 3.18.2+.
+cmake_minimum_required(VERSION 3.18.2)
+
+# Additional include directories for CMake utils.
+set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}/cmake)
```

Comment:
This is setting CMAKE_MODULE_PATH, overriding whatever was there before. Its often better to *append* to this variable:

list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}/cmake)

Reply:
Good spot, it _was_ correct at one point, but I must have splatted it during an edit

Reply:
Fixed in 2fd20e75